component/src/main/java/org/wso2/extension/siddhi/io/http/sink/HttpSink.java

```diff
@@ -127,6 +134,14 @@
                         type = {DataType.STRING},
                         optional = true,
                         defaultValue = " "),
+                @Parameter(
+                        name = "encode.payload",
+                        description = "This defines whether the body of the payload of http message should be encoded" +
+                                " or not. This is useful when sending form data as url encoded values." +
+                                " Note that this can be used only when the mapper type is keyvalue.",
+                        type = {DataType.BOOL},
+                        optional = true,
+                        defaultValue = "false"),
```

Comment:
Shall we remove this property and always encode the keyvalue ?

Reply:
+1 
Done by 92d9c4a7865aedf2fcfbe350231a90b7ed67c4ca